### ISSUE TITLE
Add `params_to_tune` for exponential smoothing models

### ISSUE THREAD

**Issue by Mr-Geekman** [tinkoff-ai/etna#1195]
### 🚀 Feature Request

Add `params_to_tune` for exponential smoothing models.

### Proposal

Sources:
- pycaret
  - [exponential smoothing](https://github.com/pycaret/pycaret/blob/master/pycaret/containers/models/time_series.py#L844)
  - [ets](https://github.com/pycaret/pycaret/blob/master/pycaret/containers/models/time_series.py#L989)
- [kats](https://github.com/facebookresearch/Kats/blob/992b95e1a8c34402756ffea2376189568de85a2c/kats/utils/parameter_tuning_utils.py#L264)

Suggested grid for `HoltWintersModel`:
```
{
    "trend": CategoricalDistribution(["add", "mul", None]),
    "damped_trend": CategoricalDistribution([False, True]),
    "seasonal": CategoricalDistribution(["add", "mul", None]),
    "use_boxcox": CategoricalDistribution([False, True]),
}
```

Suggested grid for `HoltModel`:
```
{
    "exponential": CategoricalDistribution([False, True]),
    "damped_trend": CategoricalDistribution([False, True]),
    "use_boxcox": CategoricalDistribution([False, True]),
}
```

Suggested grid for `SimpleExpSmoothingModel`:
```
{
    "use_boxcox": CategoricalDistribution([False, True]),
}
```

To discuss:
- Can we do something with `seasonal_periods` parameter in `HoltWintersModel`? The problem is that it depends on the data. For example, we can just keep the value that was put by the user and don't change it.
- What should we do about `use_boxcox=True` if there are negative values in data? In this case the model will fail probably.
- What should we do about `trend="mul"` if there are negative values in data?
- Should we touch smoothing levels during tuning?

### Test cases

Look at https://github.com/tinkoff-ai/etna/issues/1184.

**Comment by martins0n** [comment]
* `seasonal_periods` - we can skip this 
* In case of negative values I guess user could add transformers or change default grid. So it's ok to leave it this way.
* trend="mul" - Do we really have any problems here?
* Should we touch smoothing levels during tuning? - It's turning by default as I could se

**Comment by Mr-Geekman** [comment]
Closed by #1209.
